### PR TITLE
feat: add Single App Counter

### PR DESCRIPTION
**Pull request by LProulx86**
https://github.com/unoplatform/uno.samples-private/issues/146#top

Remove old Counter App
Add new Single Counter App

### CODE REVIEW

File: reference/Counter/XAML-MVUX/Counter/Counter.csproj [hunk 1]
```diff
@@ -1,67 +1,49 @@
...
-
-  <ItemGroup>
-    <UnoImage Include="Assets\**\*.svg" />
-    <UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+    <None Remove="Assets\logo.svg" />
   </ItemGroup>
-
   <ItemGroup>
-    <Content Remove="Assets\logo.svg" />
+    <UnoImage Include="Assets\logo.svg" />
```
Comment (eriklimakc) [inline reference/Counter/XAML-MVUX/Counter/Counter.csproj:46]: @LProulx86 We are missing the Assets folder as well as the logo.svg file.

image

File: reference/Counter/XAML-MVUX/Counter/MainPage.xaml.cs
Comment (eriklimakc) [inline reference/Counter/XAML-MVUX/Counter/MainPage.xaml.cs:1]: I also only see deletions of MainPage.xaml.cs but no new ones.

Comment (eriklimakc) [reply reference/Counter/XAML-MVUX/Counter/MainPage.xaml.cs:1]: Same for MainViewModel.cs and MainModel.cs

File: reference/Counter/XAML-MVUX/Counter/Counter.csproj (hunk 1 above)
Comment (LProulx86) [reply reference/Counter/XAML-MVUX/Counter/Counter.csproj:46]: I think it came from the git manipulation. I know it was there when I create it.

**Comment by nickrandolph** [comment]
Please make sure the .vscode folder is checked in for each sample - this is required for vscode support for debugging wasm and mobile targets

**Review by nickrandolph — APPROVED** [review]

File: reference/Counter/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Counter App
+
+Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.
+This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter)
```
Comment (agneszitte) [inline reference/Counter/README.md:6]: We are already in the source here no need for this link in the ReadMe

File: reference/Counter/counterApp.png
Comment (agneszitte) [inline reference/Counter/counterApp.png:1]: Can we have this image in a doc/assets folder please

**Pull request merged**